# SPDX-FileCopyrightText: 2020-2021 Smart Robotics Lab, Imperial College London
# SPDX-FileCopyrightText: 2020-2021 Sotiris Papatheodorou
# SPDX-License-Identifier: CC0-1.0

# Habitat-ROS node settings

# Sensor settings
width: 640
height: 480
near_plane: 0.1
far_plane: 20.0
fx: 525.0
fps: 30

# Other settings
scene_file: "habitat_sim/data/scene_datasets/habitat-test-scenes/skokloster-castle.glb"
#scene_file: "~/Documents/Datasets/Matterport3D/v1/tasks/mp3d_habitat/mp3d/1LXtFkjw3qL/1LXtFkjw3qL.glb"
enable_semantics: false
visualize_semantics: false
#initial_T_WB: [3, -20, 2] # Position only
#initial_T_WB: [0, 0, 0.707, 0.707] # Orientation only
#initial_T_WB: [0, -20, 2, 0, 0, 0.707, 0.707] # Position and orientation
#initial_T_WB: [0, -1, 0, 3, 1, 0, 0, -20, 0, 0, 1, 1, 0, 0, 0, 1] # 4x4 matrix

# Simple MAV simulator settings
habitat_ros_mav_sim:
    sim_freq: 60 # Hz
    a_max: [1.0, 1.0, 0.5]
    w_max: [0.1, 0.1, 0.05]

